rskj-core/src/main/java/co/rsk/net/NodeBlockProcessor.java

```diff
@@ -104,7 +104,8 @@ public void processNewBlockHashesMessage(@Nonnull final MessageChannel sender, @
                 .filter(b -> !hasBlock(b.getBytes()))
                 .forEach(
                         b -> {
-                            sender.sendMessage(new GetBlockMessage(b.getBytes()));
+                            final GetBlockMessage message1 = new GetBlockMessage(b.getBytes());
```

Comment:
Please undo this and all below at commit 6a9f1b5e41774001845bf8c3e63c3ea14fce36a8